lib/nylas/resources/notetakers.rb

```diff
@@ -0,0 +1,145 @@
+# frozen_string_literal: true
+
+require_relative "resource"
+require_relative "../handler/api_operations"
+
+module Nylas
+  # Nylas Notetaker API
+  class Notetakers < Resource
+    include ApiOperations::Get
+    include ApiOperations::Post
+    include ApiOperations::Put
+    include ApiOperations::Delete
+    include ApiOperations::Patch
+
+    # Return all notetakers.
+    #
+    # @param identifier [String, nil] Grant ID or email account to query.
+    # @param query_params [Hash, nil] Query params to pass to the request.
+    # @return [Array(Array(Hash), String, String)] The list of notetakers, API Request ID, and next cursor.
+    def list(identifier: nil, query_params: nil)
+      path = if identifier
+               "#{api_uri}/v3/grants/#{identifier}/notetakers"
+             else
+               "#{api_uri}/v3/notetakers"
+             end
+
+      get_list(
+        path: path,
+        query_params: query_params
+      )
+    end
+
+    # Return a notetaker.
+    #
+    # @param notetaker_id [String] The id of the notetaker to return.
+    # @param identifier [String, nil] Grant ID or email account to query.
+    # @param query_params [Hash, nil] Query params to pass to the request.
+    # @return [Array(Hash, String)] The notetaker and API request ID.
+    def find(notetaker_id:, identifier: nil, query_params: nil)
+      path = if identifier
```

Comment:
Should we make this path construction into a ternary?

Reply:
Agreed, done!